cmd/tika/main.go

```diff
@@ -54,12 +54,11 @@ const (
 
 // Command line flags.
 var (
-	downloadVersion = flag.String("download_version", "", "Tika Server JAR version to download. If -serverJAR is specified, it will be downloaded to that location, otherwise it will be downloaded to your working directory. If the JAR has already been downloaded and has the correct MD5, this will do nothing. Valid versions: 1.14.")
```

Comment:
I'd prefer to keep this flag, in case we have more than one version later. That also makes it nicer for people who are already using `download_version` with an old Tika version (they'd get an error it's not supported anymore).

Reply:
I reverted this flag and support only v1.19